{"name": "Non auth user data update failed test", "status": "passed", "start": 1727809287774, "stop": 1727809288403, "uuid": "a1c071c0-7135-4cae-90e2-6f100207b700", "historyId": "207907099f99c9c8590beda7b51e8b3e", "testCaseId": "207907099f99c9c8590beda7b51e8b3e", "fullName": "tests.test_update_user_data.TestUpdateUserData#test_update_user_data_non_auth_error", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_update_user_data"}, {"name": "subSuite", "value": "TestUpdateUserData"}, {"name": "host", "value": "WIN-B95O9QQ3RDO"}, {"name": "thread", "value": "23340-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_update_user_data"}]}